README.md

```diff
@@ -13,128 +13,74 @@
+### Why should I use SwiftCurrent?
+Architectural patterns and libraries that attempt to create a separation between views and workflows already exist. However, SwiftCurrent is different. We took a new design approach that focuses on
 
-## Interested but you need SwiftUI support?
+- **A Developer Friendly API**: The library was built with developers in mind. It started with a group of developers talking about the code experience they desired. Then the library team took on whatever complexities were necessary to bring them that experience.
+- **Compile-time safety**: We tell you at compile time everything we can so you know things will work.
+- **Minimal Boilerplate**: We have hidden this as much as possible. We hate it as much as you do and are constantly working on cutting the cruft.
 
-[We're working on it now!](https://github.com/wwt/SwiftCurrent/milestone/2)
-
-If you would like to try the beta release, please install the `BETA_SwiftCurrent_SwiftUI` product in SPM or the `BETA_SwiftUI` sub spec in CocoaPods.  For more detailed steps, [see our installation instructions](https://wwt.github.io/SwiftCurrent/installation.html).  See [Getting Started with SwiftUI](https://wwt.github.io/SwiftCurrent/getting-started-with-swiftui.html) for a quick tutorial.  To see the example app for SwiftUI, clone our repo, open `SwiftCurrent.xcworkspace`, target the `SwiftUIExample` scheme, and run. The [example app has a README](https://github.com/wwt/SwiftCurrent/blob/main/ExampleApps/SwiftUIExample/README.md) that details interesting usages.
-
-In order to use the library with SwiftUI, your minimum targeted versions must meet: iOS 14.0, macOS 11, tvOS 14.0, or watchOS 7.0.
-
-For us, beta means that the API may change without warning until the full release.  However, we expect bugs to be at a minimum and documentation to be true and accurate.
+#### From there, we created a library that:
+- **Isolates your views**:  You can design your views so that they are unaware of the view that will come next.
+- **Easily reorders views**: Changing view order is as easy as ⌘+⌥+\[ (moving the line up or down)
+- **Composes workflows together**: Create branching flows easily by joining workflows together.
+- **Creates conditional flows**: Make your flows robust and handle ever-changing designs. Need a screen only to show up sometimes? Need a flow for person A and another for person B? We've got you covered.
 
 # Quick Start
-
-This quick start uses SPM, but for other approaches, [see our installation instructions](https://wwt.github.io/SwiftCurrent/installation.html).
-
-## UIKit
+Why show a quick start when we have an example app? Because it's so easy to get started, we can just drop in 2 code snippets and you're ready to go! This quick start uses Swift Package Manager and SwiftUI, but for other approaches, [see our installation instructions](https://wwt.github.io/SwiftCurrent/installation.html).
```

Comment:
```suggestion
Why show a quick start when we have an example app? Because it's so easy to get started, we can drop in 2 code snippets, and you're ready to go! This quickstart uses Swift Package Manager and SwiftUI, but for other approaches, [see our installation instructions](https://wwt.github.io/SwiftCurrent/installation.html).
```